README.md

```diff
@@ -202,6 +203,32 @@ To send an email, you just need to call the static class:
     Mailer.Send(toAddress, subject, body);
 ```
 
+## Globalization
+
+To globalize our application, we are going to add the localization culture core package to the project:
```

Comment:
To globalize the application is necessary to add the localization culture core package to the project, to do this just run: